kolla-ansible/ansible/roles/triliovault/tasks/start_wlm_cron_service.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: Stop wlm_cron service
+  vars:
+    triliovault_wlm_cron: "{{ triliovault_services['triliovault-wlm-cron'] }}"
+  become: true
+  kolla_docker:
+    action: "stop_container"
+    common_options: "{{ docker_common_options }}"
+    name: "{{ triliovault_wlm_cron.container_name }}"
+  when: inventory_hostname in groups[triliovault_wlm_cron_group]
```

Comment:
This will stop wlm cron service on wlm-cron group host only.
We need to stop wlm service on all hosts.

Reply:
It will start on all hosts, validated on **Kolla-Ansible Wallaby Ubuntu 20.04 setup**